### PR TITLE
feat: Create wg-qa initial README.md

### PR DESCRIPTION
**Pull request by SlackedMarcel**
wg-qa initial PR

### CODE REVIEW

File: wg-qa/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Releases WG
```
Comment (sofianguy) [inline wg-qa/README.md]: ~Releases WG~ --> QA Working Group

File: wg-qa/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+| Name |
+| -----------------|
+| Maria (Pending)  |
+| Keeley (Pending) |
+| David            |
+| Alexey           |
+| Sofia            |
+
+## Chair
```
Comment (sofianguy) [inline wg-qa/README.md]: For this section, see [Releases WG readme](https://github.com/electron/governance/blob/master/wg-releases/README.md) for an example on how to format the Membership section. We would like all the WGs to have consistent READMEs 🙂

File: wg-qa/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Areas of Responsibility
+
+* Strongly advises the stable release.
+* Long-term would oversee test automation and greenlight the stable release.
+
+## Meeting Schedule
+
+* **Sync Meeting** Every 4 weeks, wed 9:30am-PST (https://calendar.google.com/event?action=TEMPLATE&tmeid=NGp0NGU4OGtiYzExOGVvbWJkcmNpNGlyY2hfMjAxOTA5MjVUMTYzMDAwWiBjYzM3dmx1c2w2Z3Y2dnVvbjhrZGNqYzc3a0Bn&tmsrc=cc37vlusl6gv6vuon8kdcjc77k%40group.calendar.google.com&scp=ALL)
+* **Sync Meeting** 45 min every Wednesday at [23:00 UTC](https://duckduckgo.com/?q=23%3A00+UTC&ia=answer)
```
Comment (sofianguy) [inline wg-qa/README.md]: copy-pasta error

File: wg-qa/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+* **Major Release Cadence Meeting** 1 hour every Thursday at [00:00AM UTC](https://duckduckgo.com/?q=00%3A00+UTC&ia=answer)
```
Comment (sofianguy) [inline wg-qa/README.md]: copy-pasta error

File: wg-qa/README.md (hunk 2 above)
Comment (SlackedMarcel) [reply wg-qa/README.md]: Fixed

File: wg-qa/README.md (hunk 1 above)
Comment (SlackedMarcel) [reply wg-qa/README.md]: duh.... fixed

File: wg-qa/README.md (hunk 3 above)
Comment (SlackedMarcel) [reply wg-qa/README.md]: fixed... sigh

File: wg-qa/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+## Membership
+
+| Avatar| Name | Role | Time Zone |
+| ----------------- | ----------------- | ----------------- | ----------------- |
+| <img src="https://github.com/MariaBeliaeva.png" width=100 alt="@MariaBeliaeva">  | Maria Beliaeva [@MariaBeliaeva](https://github.com/MariaBeliaeva)| **Chair** | PT (San Francisco) |
+| <img src="https://github.com/VerteDinde.png" width=100 alt="@VerteDinde">  | Keeley Hammond [@VerteDinde](https://github.com/VerteDinde) | Member | PT (Portland) |
+| <img src="https://github.com/cruzerld.png" width=100 alt="@cruzerld">  | David Cruz [@cruzerld](https://github.com/cruzerld) | Member | ET (New York City) |
+| <img src="https://github.com/alexeykuzmin.png" width=100 alt="@alexeykuzmin">  | Alexey Kuzmin [@alexeykuzmin](https://github.com/alexeykuzmin) | Member | CET (Prague) |
+| <img src="https://github.com/sofianguy.png" width=100 alt="@sofianguy"> | Sofia Nguy [@sofianguy](https://github.com/sofianguy) | Member | PT (San Francisco) |
+
```
Comment (sofianguy) [inline wg-qa/README.md]: You're forgetting yourself, @SlackedMarcel !

Also add Andrea

File: wg-qa/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+* Releases WG
+
+## Areas of Responsibility
+
+* Strongly advises the stable release.
+* Long-term would oversee test automation and greenlight the stable release.
+
+## Meeting Schedule
+
+* **Sync Meeting** Every 4 weeks, wed 9:30am-PST (https://bit.ly/2n31SNx)
```
Comment (sofianguy) [inline wg-qa/README.md]: nit: capital W for Wed

**Comment by sofianguy** [comment]
Added @jkleinsc to reviewers, as he is the current Releases WG chair.

File: wg-qa/README.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+| <img src="https://github.com/cruzerld.png" width=100 alt="@cruzerld">  | David Cruz [@cruzerld](https://github.com/cruzerld) | Member | ET (New York City) |
+| <img src="https://github.com/alexeykuzmin.png" width=100 alt="@alexeykuzmin">  | Alexey Kuzmin [@alexeykuzmin](https://github.com/alexeykuzmin) | Member | CET (Prague) |
+| <img src="https://github.com/sofianguy.png" width=100 alt="@sofianguy"> | Sofia Nguy [@sofianguy](https://github.com/sofianguy) | Member | PT (San Francisco) |
+
+
+
+## Parent Group
+
+* Releases WG
+
```
Comment (sofianguy) [inline wg-qa/README.md]: Above Areas of Responsibility, we need to add our OKRs from the post-summit meeting. Like so:

```
## Current Objective and Key Results
**Objective:**

Electron stable releases should be of high quality and easy for apps to upgrade.

**Key Results:**
* Measure the average or cumulative delta between an electron stable release, and the AFP participants shipping that same stable release in their GA product.
* Measure the average or cumulative delta between an electron stable release, and the AFP participants shipping that same stable release in their canary product.
* Reduce the size of outstanding AFP-raised issues at the time of stable.
* Reduce the number of _would-be blockers_ discovered _after_ a stable ship.

```

Comment (groundwater) [reply wg-qa/README.md]: I would like to see these integrated into the group. Remember, the KRs are not _must haves_ they are just guides to help tell if you actually hit your objective.

May also be worth including a KR about growing out the working group.

File: wg-qa/README.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+## Areas of Responsibility
+
+* Strongly advises the stable release.
+* Long-term would oversee test automation and greenlight the stable release.
```
Comment (groundwater) [inline wg-qa/README.md]: Is the AFP going to live under this WG?

**Comment by groundwater** [comment]
This is great work @SlackedMarcel. Sorry the _request changes_ button produces a giant red assault on the senses.

File: wg-qa/README.md (hunk 8 above)
Comment (sofianguy) [reply wg-qa/README.md]: I recall some discussion at Summit about AFP falling under QA WG, but it also make senses under Releases WG. However, QA is under Releases so maybe? 🤔

File: wg-releases/wg-qa/meeting-notes/2019-9-18.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+Attendees: 
+
+* Link to pre-summit session notes:
+    * [CI, Build, & Test pre-summit](https://hackmd.io/LJZ9D222RlulmL5clYisqA)
+* Link to summit session notes:
+    * [Testing session](http://bit.ly/ems-testing)
+    * [CI & Build](http://bit.ly/ems-ci)
+* Link to summit session photos:
+    * https://photos.app.goo.gl/eHpRPxJ3wcLH87uk6
+
```
Comment (sofianguy) [inline wg-releases/wg-qa/meeting-notes/2019-9-18.md]: Can we delete this from our meeting notes? The governance repo is readable to the general public and we don't want these summit links to be public. The rest of the notes look OK for public.

File: wg-qa/README.md (hunk 7 above)
Comment (SlackedMarcel) [reply wg-qa/README.md]: Added KRs, and with Maria added a growth KR

File: wg-releases/wg-qa/meeting-notes/2019-9-18.md (hunk 9 above)
Comment (SlackedMarcel) [reply wg-releases/wg-qa/meeting-notes/2019-9-18.md]: Done!

**Pull request merged**